subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
                 </a>
             </div></li>
             <li><a href="/userguide/getting_started.html">Getting Started</a></li>
-        </ul>
-
-        <h3 id="using-gradle">Using Gradle</h3>
-        <ul>
-            <li><a href="/userguide/installation.html">Installing Gradle</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading Gradle</a>
+            <li><a href="/userguide/installation.html">Installation</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#upgrading-gradle" aria-expanded="false" aria-controls="upgrading-gradle">Upgrading &amp; Migrating</a>
```

Comment:
I mainly wanted to slim down the top set of links, so have tried combining upgrading and migrating into a single heading. To be honest, I feel like upgrading from one version of Gradle to another is a migration anyway.

Reply:
I think we talked about this before. I'm not super keen on combining things just to shorten the number of lines, but this isn't the worst violation. If you feel strongly that this is important we can keep it, but I weakly think they ought to be separate.

Reply:
I don't feel particularly strongly. I may try a couple of other things.

Reply:
My first question was 'Migrating from what to what?'. I had to look at the change below to figure it out.  If they stay in one heading can we say something like 'Migrating from other build tools?'